### ISSUE TITLE
After getting back from any click(link) , we should be on same position but we get to start of webpage 

### ISSUE THREAD

**Issue by knightmate** [commudle/commudle-ng#101]
If we click on any project ,when we get back from that project link we get navigated at starting of webpage not on same project section.

**Comment by arpansac** [comment]
This is going to be an interesting thing to solve!

**Comment by Ashwin-op** [comment]
Maybe we can do something like,

During page unload, get the scroll position and store it in session storage (We should use session storage instead of local storage since session storge affects only the current tab while local storage shared between all tabs and windows of the same origin).

And this can be done per page (by appending pathname maybe?) so other pages in the same origin don't get affected.

**Comment by arpansac** [comment]
Yup, I think that should be a smart way of doing it, since the complete page is loaded via ajax.
Plus since we have SSR enabled on the project, we could choose some of the request to be pre-loaded before the page is rendered.

As for the specific tab, I'm generating a unique app_id for every refresh that takes place (being sent from the server to front end), or we could find some unique id for that tab too
